README.md

```diff
@@ -0,0 +1,17 @@
+# Contacts Manager web app
+
+## Requirements:
```

Comment:
You can create a Virtual Machine with something like [venv](https://docs.python.org/3/library/venv.html) that allows you to bundle the packages into a `requirements.txt` file that will automatically install the packages a user will need to run your programme. (You can see an example in my Python Tic Tac Toe [README](https://github.com/codeidoscope/noughts-and-crosses) and [requirements.txt](https://github.com/codeidoscope/noughts-and-crosses/blob/master/requirements.txt).

The nice thing about this approach is that you can specify which version of the package you want to use, so there's fewer risks of the programme not working if a library's been updated or your code doesn't support backward compatibility. It will also ensure that people who have Python 2.7 installed on their machine can very quickly and simply run your Python 3 programme in a VM instead of faffing with installing Python 3.

Reply:
Awesome, I'll try to add it! Thank you Marion!